core/cache/redisBackend.go

```diff
@@ -0,0 +1,378 @@
+/**
+ * @TODO:
+ * - write documentation
+ */
+
+package cache
+
+import (
+	"bytes"
+	"encoding/gob"
+	"fmt"
+	"regexp"
+	"runtime"
+	"time"
+
+	"flamingo.me/flamingo/v3/framework/flamingo"
+	"github.com/gomodule/redigo/redis"
```

Comment:
any reasons why this package?

I found also: https://github.com/go-redis/redis   which looks nice from the API

(Did not do a compare - just want to know)

Reply:
because it is already used here: https://github.com/i-love-flamingo/flamingo/blob/master/core/healthcheck/domain/healthcheck/redisSession.go#L3

and i do not want to integrate a second library for the same thing...